actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadManagementActor.java

```diff
@@ -378,6 +381,7 @@ private void uploadCsvToDB(
       String objectType,
       String requestedBy,
       String rootOrgId) {
+    ProjectLogger.log("BulkUploadManagementActor:uploadCsvToDB method start", LoggerEnum.INFO);
```

Comment:
From:
"BulkUploadManagementActor:uploadCsvToDB method start"

To:
"BulkUploadManagementActor: uploadCsvToDB called."